lib/js-yaml/dumper.js

```diff
@@ -12,7 +12,7 @@ var _hasOwnProperty = Object.prototype.hasOwnProperty;
 
 var CHAR_TAB                  = 0x09; /* Tab */
 var CHAR_LINE_FEED            = 0x0A; /* LF */
-var CHAR_CARRIAGE_RETURN      = 0x0D; /* CR */
+// var CHAR_CARRIAGE_RETURN      = 0x0D; /* CR */ // unused
```

Comment:
No reason to keep it in any form here if it's not in use. It can be just re-added in future if needed.